adapters/huaweiads/huaweiads.go

```diff
@@ -788,39 +778,54 @@ func convertCountryCode(country string) (out string) {
 	return defaultCountryName
 }
 
+func getCountryCodeFromMCC(MCC string) (out string) {
+	var country_mcc = strings.Split(MCC, "-")[0]
+	intVar, err := strconv.Atoi(country_mcc)
+
+	if err != nil {
```

Comment:
Can you please add some unit tests for this function as well. There doesn't seem to be any coverage for this function